test_sets/mobile_api_versioning.txt

```diff
@@ -0,0 +1,2 @@
+./test_scripts/SDL4_6/MobileVersioning/VersionNegotiation/001_register_legacy_app.lua
+./test_scripts/SDL4_6/MobileVersioning/VersionNegotiation/002_register_app_on_legacy_module.lua
```

Comment:
Please add new empty line in the end. For now only one script is executed by set running.